dev-db/sqlitebrowser/sqlitebrowser-3.9.1.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="DB Browser for SQLite"
+DESCRIPTION="DB Browser for SQLite is a high quality, visual, open source tool \
+to create, design, and edit database files compatible with SQLite."
+HOMEPAGE="http://sqlitebrowser.org/"
+COPYRIGHT="2012-2016 Martin Kleusberg"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/sqlitebrowser/sqlitebrowser/archive/v3.9.1.tar.gz"
+CHECKSUM_SHA256="d0d2e06a69927ba1d0b955f3261ce70c61befc5bd5ddaa06752dae8bb4219ed8"
+PATCHES="sqlitebrowser-3.9.1.patchset"
+ADDITIONAL_FILES="sqlitebrowser.rdef"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	sqlitebrowser$secondaryArchSuffix = $portVersion
+	app:sqlitebrowser$secondaryArchSuffix = $portVersion
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libQtCore$secondaryArchSuffix
+	lib:libQtGui$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	libqt4${secondaryArchSuffix}_devel
+	devel:libsqlite3$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:qmake
+	cmd:make
+	cmd:g++$secondaryArchSuffix
+"
+
+BUILD()
+{
+	qmake -o Makefile $sourceDir/sqlitebrowser.pro
+	make $jobArgs
+
+	rc $portDir/additional-files/sqlitebrowser.rdef -o $sourceDir/src/sqlitebrowser.rsrc
+
+	xres -o $sourceDir/src/sqlitebrowser $sourceDir/src/sqlitebrowser.rsrc
+	mimeset -f $sourceDir/src/sqlitebrowser
+}
+
+INSTALL()
+{
+	make $jobArgs install
+	mkdir -p $appsDir
+	mv $sourceDir/src/sqlitebrowser $appsDir/SQLiteBrowser
```

Comment:
I'm not entirely sure how to change the install directory. I think it would require tinkering with `qmake` and I'm not very familiar with it. I think this suffices for now since SQLite Browser only has an executable (no icons or other files).